### PR TITLE
[MU4] Fix #11132: Note input still "enabled" in publish

### LINKED ISSUE #11132: [MU4 Issue] Note input still "enabled" in publish
**Describe the bug**
When in note input mode, you can still input notes in publish (but not accidentals).

**To Reproduce**
1. Click on score tab.
2. Enable note input mode (i.e. by pressing 'n' so that the pencil icon is selected).
3. Click on publish tab. 
4. Add notes (but you can't raise it semitone up or down).

**Expected behavior**
I don't think we should be able to add notes in publish.

**Platform information**
 - Windows

**Additional Context:**
Not sure if this is bug or feature. What should be editable in publish tab? Thanks!

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/11132

### CODE REVIEW

File: src/notation/view/notationviewinputcontroller.cpp [hunk 1]
```diff
@@ -44,7 +44,7 @@ void NotationViewInputController::init()
 {
     m_possibleZoomPercentages = configuration()->possibleZoomPercentageList();
 
-    if (dispatcher() && !m_readonly) {
+    if (dispatcher() && editingAllowed()) {
```
Comment (cbjeukendrup) [inline src/notation/view/notationviewinputcontroller.cpp:47]: Shouldn't those zoom/viewmode actions be available in Publish mode too?

**Comment by igorkorsukov** [comment]
Could you resolve the conflicts

**Comment by RobFog** [comment]
Was this pull request merged manually here: https://github.com/RomanPudashkin/MuseScore/commit/abd64f5e98c292fa9ac3adcabfbd62c415cb7faa ? If so, it should be closed?

**Comment by RomanPudashkin** [comment]
> Was this pull request merged manually here: [RomanPudashkin@abd64f5](https://github.com/RomanPudashkin/MuseScore/commit/abd64f5e98c292fa9ac3adcabfbd62c415cb7faa) ? If so, it should be closed?

It wasn't, but this fix should be revised anyway

**Pull request closed without merging**